rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -1,12 +1,26 @@
 function (user, context, callback) {
-    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (dev)';
+    var rangeCheck = require('range_check');
+
+    var AUTHENTICATOR_LABEL = 'MOJ Analytical Platform (Alpha)';
```

Comment:
Shouldn't this be `dev`? Was this copied from `dev`?

Reply:
ah, it does say dev on dev, so this I must have made an error here

Reply:
I see that dev got auto-deployed (triggered by other changes) the old version of this a couple of days ago. But yes it said Alpha before that. I'll push a fresh version that is a copy from alpha but with this word changed.